scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixEdit.java

```diff
@@ -0,0 +1,13 @@
+package scalafix.interfaces;
+
+public interface ScalafixEdit {
```

Comment:
Even if a bit pedantic, 
```suggestion
public interface ScalafixTextEdit {
```
would be more consistent